ihatemoney/forms.py

```diff
@@ -146,6 +146,8 @@ class BillForm(FlaskForm):
     what = StringField(_("What?"), validators=[DataRequired()])
     payer = SelectField(_("Payer"), validators=[DataRequired()], coerce=int)
     amount = CalculatorStringField(_("Amount paid"), validators=[DataRequired()])
+    external_link = URLField(_("External link"), validators=[Optional(
+    )], description="A link to an external document, related to this bill")
```

Comment:
The description should probably be translatable too.

Reply:
I can easily add a `_("A link to an external document, related to this bill")` but I don't know how to generate correctly the translation even after a `make update-translation` with some changes in messages.po. Am I missing something ?

Reply:
`make update-translations` will update the strings, and then the translation process is done via Weblate (http://hosted.weblate.org/). This is currently undocumented, so you didn't have any way to know, sorry.